bindings/jaxrs/src/main/java/org/commonjava/indy/core/bind/jaxrs/util/TransferStreamingOutput.java

```diff
@@ -49,19 +51,18 @@ public TransferStreamingOutput( final InputStream stream )
                                     + IndyMetricsNames.METER ), timers = @MetricNamed( name =
                     IndyMetricsBindingsNames.METHOD_TRANSFERSTREAMING_WRITE + IndyMetricsNames.TIMER ) ) )
     public void write( final OutputStream out )
-        throws IOException, WebApplicationException
+            throws IOException, WebApplicationException
     {
-        try
+        synchronized ( transfer )
```

Comment:
transfer.lockRead()?

Reply:
See my comment in the ctor, above.

I'm not sure what synchronizing on the transfer here will do beyond that.